backend/de.metas.externalsystem/src/main/sql/postgresql/system/80-de.metas.externalsystem/5645370_sys_gh13198_External_System_Auth_Token.sql

```diff
@@ -0,0 +1,83 @@
+-- 2022-06-28T13:55:04.441Z
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541464,'S',TO_TIMESTAMP('2022-06-28 16:55:04','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','EXTERNAL_SYSTEM_AUTHORIZATION_TOKEN',TO_TIMESTAMP('2022-06-28 16:55:04','YYYY-MM-DD HH24:MI:SS'),100,'2ac8d2ac2c0d485c970db5d96f89d809')
+;
+
+-- 2022-06-30T14:59:46.179Z
+UPDATE AD_SysConfig SET ConfigurationLevel='S', Description='This is a sys config with an authorization token that is used to check that metasfresh has the appropriate authorization token.', Name='de.metas.externalsystem.externalservice.authorization.authToken',Updated=TO_TIMESTAMP('2022-06-30 17:59:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541464
+;
+
+-- 2022-07-01T14:27:12.019Z
+INSERT INTO AD_UserGroup (AD_Client_ID,AD_Org_ID,AD_UserGroup_ID,Created,CreatedBy,IsActive,Name,Updated,UpdatedBy) VALUES (1000000,1000000,540005,TO_TIMESTAMP('2022-07-01 17:27:12','YYYY-MM-DD HH24:MI:SS'),100,'Y','API-Setup',TO_TIMESTAMP('2022-07-01 17:27:12','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
please add an informative description to this group and also to the respective AD_SysConfig